server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,178 @@
+df5 = pd.read_json(API_HOST + query_string)
+reqAgency = df5.groupby(['agency_name'])['counts'].sum()
+
+# Request Share by Agency Pie Chart
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=agency_name&filter=created_date>=2016-01-01"
+df5 = pd.read_json(API_HOST + query_string)
+df5 = df5.groupby(['agency_name'])['counts'].sum().to_frame()
+df5.sort_values('counts', ascending=False, inplace=True)
+df5.loc['Others'] = df5[4:].sum()
+df5.sort_values('counts', ascending=False, inplace=True)
+df5 = df5[:5]
+df5.index = df5.index.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))
+
+shareReqByAgencyPieChart = px.pie(
+    df5,
+    names=df5.index,
+    values='counts',
+    labels=LABELS,
+    hole=.3,
+    title="Total Requests by Agency",
+)
+shareReqByAgencyPieChart.update_layout(margin=dict(l=100, r=100, b=100, t=100))
+
+# Request Type by Source Bar Chart
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=source_name&filter=created_date>=2016-01-01"
+df6 = pd.read_json(API_HOST + query_string)
+df6 = df6.groupby(['source_name'])['counts'].sum().to_frame()
+df6.sort_values('counts', ascending=False, inplace=True)
+df6.loc['Others'] = df6[4:].sum()
+df6.sort_values('counts', ascending=False, inplace=True)
+df6 = df6[:5]
+reqSourceBarchart = px.bar(
+    df6,
+    y=df6.index,
+    x='counts',
+    title="Total Requests by Source",
+    orientation='h'
+)
+
+# Median Request Days to Close Box Plot
+stas_df = pd.read_json(API_HOST + '/types/stats')
+stas_df = stas_df.sort_values('median', ascending=False)
+medDaysToCloseBoxPlot = go.Figure()
+medDaysToCloseBoxPlot.add_trace(
+    go.Box(
+        y=stas_df.type_name,
+        q1=stas_df['q1'],
+        median=stas_df['median'],
+        q3=stas_df['q3'],
+        marker_color='#29404F',
+        fillcolor='#E17C05',
+    )
+)
+medDaysToCloseBoxPlot.update_xaxes(
+
+    dtick=5
+)
+medDaysToCloseBoxPlot.update_layout(
+    title="Total Median Days to Close by Type",
+)
+
+# Day of Week Bar Chart:
+start_date = datetime.date.today() - datetime.timedelta(days=30)
+end_date = datetime.date.today() - datetime.timedelta(days=1)
+query_string = f"/reports?filter=created_date>={start_date}&filter=created_date<={end_date}"  # noqa
+print(" * Downloading data for dataframe")
+df = pd.read_json(API_HOST + query_string)
+df['created_date'] = pd.to_datetime(df['created_date'])
+dow_df = df.groupby(['created_date']).agg('sum').reset_index()
+dow_df['day_of_week'] = dow_df['created_date'].dt.day_name()
+numReqByDayOfWeekBarChart = px.bar(
+    dow_df,
+    x="day_of_week",
+    y="counts",
+    labels=LABELS,
+)
+numReqByDayOfWeekBarChart.update_xaxes(categoryorder='array', categoryarray= ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"])
+
+# Total Request by NC
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=council_name&filter=created_date>=2016-01-01"
```

Comment:
We're already downloading this in line 22-23.

Reply:
Thanks for the catch! Re-used the previous df.